exercicios/para-casa/sistema_de_alunas_mariana_araujo.py

```diff
@@ -0,0 +1,157 @@
+    codigo_opcao = 0
+
+    while codigo_opcao not in [1, 2, 3, 4, 5, 6]:
+        try:
+            codigo_opcao = int(input("\nEscolha uma opção:\n"
+                                    "1 - Incluir uma nova aluna\n"
+                                    "2 - Consultar lista de alunas\n"
+                                    "3 - Consultar faltas da aluna\n"
+                                    "4 - Consultar notas da aluna\n"
+                                    "5 - Consultar status de aprovação\n"
+                                    "6 - Sair do sistema\n"
+                                    "Opção: "))
+                
+            if codigo_opcao not in [1, 2, 3, 4, 5, 6]:
+                print("Opção inválida. Por favor, escolha uma opção válida (1 a 5).\n")
+        except ValueError:
+            print("Entrada inválida. Por favor, digite um número inteiro.\n")
+            
+        return codigo_opcao
+    
+def incluir_nova_aluna():
+    print('\nInsira os seguintes dados: ')
+    nome = input('Nome da aluna: ')
+    sobrenome = input('Sobrenome da aluna: ')
+    turma = input('Turma da aluna: ') 
+    lista_de_presenca = presenca(nome, sobrenome) 
+    nota_participacao = float(input('Nota de participação da aluna (0-10): ')) 
+    notas = obter_notas(nome,sobrenome)
+
+    salvar_dados_aluna(nome, sobrenome, turma, notas, lista_de_presenca, nota_participacao)
+    return incluir_nova_aluna
+
+
+def presenca(nome,sobrenome):
+    print('\nInsira os seguintes dados: ')
+    lista_de_presenca = []
+    for i in range(1,6):
+        while True:
+            contagem = input(f'Aluna compareceu a aula do dia {i}? True/False: ')
+            if contagem == 'True':
+                lista_de_presenca.append(True)
+                break
+            elif contagem == 'False':
+                lista_de_presenca.append(False)
+                break
+            else:
+                print('Dado inválido. Por favor, insira True ou False.')
+
+    return lista_de_presenca
+
+def obter_notas(nome,sobrenome):
+    print('\nInsira os seguintes dados: ')
+    notas = []
+    for contador in range(1,4): 
+          while True: 
+            entrada = input(f'Insira a nota da aula {contador}: ') 
+            try: 
+                nota = float(entrada) 
+                notas.append(nota) 
+                break
+            except ValueError: 
+                print('Dado inválido. Por favor, insira um número válido.')
+    return notas
+            
+
+def salvar_dados_aluna(nome, sobrenome, turma, notas, lista_de_presenca, nota_participacao):
+    
+    dataset[nome,sobrenome] = { 
+        "Turma": turma,
+        "Notas": notas,
+        "Presença": lista_de_presenca,
+        "Participação": nota_participacao
+    }
+    return(print('Aluna',nome,sobrenome,'foi adicionada com sucesso!'))
+
+
+def consultar_lista_alunas():
+    lista_de_alunas = dataset.keys()
+    for item in lista_de_alunas:
+        print('.',item[0],item[1])   
```

Comment:
Experimente desempacotar a tupla para tornar o código mais claro. Ou seja, dê um nome para as variáveis que você está utilizando dentro do seu for e ao invés de utilizar os índices da tupla, você pode usar os nomes das variáveis que você criou:

**Sugestão:**
```suggestion
    for nome, sobrenome in lista_de_alunas:
        print('.', nome, sobrenome)
```